harmonica/filters/_filters.py

```diff
@@ -60,5 +60,478 @@ def derivative_upward_kernel(fft_grid, order=1):
+    da_filter = np.exp(np.sqrt(k_easting ** 2 + k_northing ** 2) * height)
+    return da_filter
+
+
+def gaussian_lowpass_kernel(fft_grid, wavelength):
+    r"""
+    Filter for gaussian low-pass in frequency domain
+
+    Return a :class:`xarray.DataArray` with the values of the frequency domain
+    filter for computing the northing derivative. The filter is built upon the
+    frequency coordinates of the passed ``fft_grid`` and is defined as follows:
+
+    .. math::
+
+        g(\mathbf{k}) = e^\frac{-|\mathbf{k}| ^ 2}{\mathbf{kw}^2}
+
+    where :math:`\mathbf{k}` is the wavenumber vector
+    (:math:`\mathbf{k} = 2\pi \mathbf{f}` where :math:`\mathbf{f}` is the
+    frequency vector) and `\mathbf{kw}` is the wavenumber of the cutoff
+    wavelength
+    :math:`\mathbf{w}` (:math:`\mathbf{kw} = \frac{2\pi} {\mathbf{w}}`).
+
+    Parameters
+    ----------
+    fft_grid : :class:`xarray.DataArray`
+        Array with the Fourier transform of the original grid.
+        Its dimensions should be in the following order:
+        *freq_northing*, *freq_easting*.
+        Use :func:`xrft.xrft.fft` and :func:`xrft.xrft.ifft` functions to
+        compute the Fourier Transform and its inverse, respectively.
+    wavelength : float
+        The cutoff wavelength in low-pass filter. It has the same units as the
+        input xarray data.
+
+    Returns
+    -------
+    da_filter : :class:`xarray.DataArray`
+        Array with the kernel for the gaussian low-pass filter in frequency
+        domain.
+
+    References
+    ----------
+    [Geosoft1999]_
+
+    See also
+    --------
+    harmonica.gaussian_lowpass
+    """
+    # Catch the dims of the Fourier transformed grid
+    dims = fft_grid.dims
+    # Grab the coordinates of the Fourier transformed grid
+    freq_easting = fft_grid.coords[dims[1]]
+    freq_northing = fft_grid.coords[dims[0]]
+    # Convert frequencies to wavenumbers
+    k_easting = 2 * np.pi * freq_easting
+    k_northing = 2 * np.pi * freq_northing
+    # Compute the filter for northing derivative in frequency domain
+    da_filter = np.exp(
+        -(k_easting ** 2 + k_northing ** 2) / (2 * (2 * np.pi / wavelength) ** 2)
+    )
+    return da_filter
+
+
+def gaussian_highpass_kernel(fft_grid, wavelength):
+    r"""
+    Filter for gaussian high-pass in frequency domain
+
+    Return a :class:`xarray.DataArray` with the values of the frequency domain
+    filter for computing the northing derivative. The filter is built upon the
+    frequency coordinates of the passed ``fft_grid`` and is defined as follows:
+
+    .. math::
+
+        g(\mathbf{k}) = 1-e^\frac{-|\mathbf{k}| ^ 2}{(2\mathbf{kw})^2}
+
+    where :math:`\mathbf{k}` is the wavenumber vector
+    (:math:`\mathbf{k} = 2\pi \mathbf{f}` where :math:`\mathbf{f}` is the
+    frequency vector) and `\mathbf{kw}` is the wavenumber of the cutoff
+    wavelength
+    :math:`\mathbf{w}` (:math:`\mathbf{kw} = \frac{2\pi} {\mathbf{w}}`).
```

Comment:
Same here regarding the description of the filter and the definition of the cutoff wavenumber.

Reply:
Yup~